DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Centre/Administrator/SearchableAdminViewModel.cs

```diff
@@ -6,15 +6,24 @@
 
     public class SearchableAdminViewModel : BaseFilterableViewModel
     {
-        public SearchableAdminViewModel(AdminUser adminUser, bool hasSuperAdminAccess, int currentAdminUserId)
+        public SearchableAdminViewModel(AdminUser adminUser, AdminUser loggedInAdminUser)
         {
             Id = adminUser.Id;
             Name = adminUser.SearchableName;
             CategoryName = adminUser.CategoryName ?? "All";
             EmailAddress = adminUser.EmailAddress;
             IsLocked = adminUser.IsLocked;
-            CanShowDeactivateAdminButton = hasSuperAdminAccess
-                                           && currentAdminUserId != adminUser.Id;
+
+            if (loggedInAdminUser.IsUserAdmin)
+            {
+                CanShowDeactivateAdminButton = adminUser.Id != loggedInAdminUser.Id;
+            }else if (loggedInAdminUser.IsCentreManager)

```

Comment:
There's a little formatting issue here, make sure you run the formatter. But besides that everything looks fine, no need to re-review.